CMakeLists.txt

```diff
@@ -460,6 +461,46 @@ if(USE_PIPELINE_EXECUTOR)
   list(APPEND RUNTIME_SRCS ${RUNTIME_PIPELINE_SRCS})
 endif(USE_PIPELINE_EXECUTOR)
 
+# Caches the build.
```

Comment:
organizational nit: can we move this into something like `cmake/ccache.cmake` and `include` it here instead?

Reply:
done